.template/addons/svgeez/bin/svg-sprite

```diff
@@ -0,0 +1,15 @@
+# Generates an SVG sprite from a folder of SVG icons.
+#
+# Uses `svgeez` gem.
+#   https://github.com/jgarber623/svgeez
+#
+# Usage
+#  -s --source:	Path to the folder of source SVGs (defaults to ./_svgeez).
+#  -d --destination:	Path to the destination file or folder (defaults to ./svgeez.svg)
+#  --with-svgo:	Optimize SVG sprite file with SVGO
+
+# Generate the sprite file which includes the `icon-` prefix
+bin/bundle exec svgeez build --source app/assets/images/icons/ --destination app/assets/images/icon.svg --with-svgo
+
+# Rename the icon.svg to icon-sprite.svg
+mv app/assets/images/icon.svg app/assets/images/icon-sprite.svg
```

Comment:
We can combine these two steps together now as `svgeez` now [supports prefix flag](https://github.com/jgarber623/svgeez#options-and-defaults)

```suggestion
# Generate the sprite file which includes the `icon-` prefix
bin/bundle exec svgeez build --prefix icon --source app/assets/images/icons/ --destination app/assets/images/icon-sprite.svg --with-svgo
```